packages/vat-data/src/kind-utils.js

```diff
@@ -21,7 +21,6 @@ export const ignoreContext =
   fn =>
   (context, ...args) =>
     fn(...args);
-// @ts-expect-error TODO statically recognize harden
```

Comment:
Yay! You got rid of this annoyance! How did you do that, and can we do it elsewhere we suffer from this annoyance? (I admit I never understood the issue that was blocking this)

Hmm. Maybe this PR does fix all those places. Do you know if there are others?

Reply:
I think this line did it https://github.com/Agoric/agoric-sdk/blob/4f51b415ed6602679c8d3c36f32ae2b13a0f061f/packages/vat-data/src/index.js#L1